### ISSUE TITLE
Face closed to the camera

### ISSUE THREAD

**Issue by diego0718** [kb22/Create-Face-Data-from-Images#1]
Good morning.

I am trying the dnn  face detector from opencv. Did you try taking a photo close to the camera?. I did this, and found more than one detection ( more than one face) and some of the bounding box coordinates are outside the image and they are not faces  ( i mean, i.e  if the images is 640*480 some bboxes coordinates are 720*680 from the top left corner)

**Comment by kb22** [comment]
No, I did not try that. Maybe that is something not built into the version of face detection model we are using here. But i can surely check and see.
Do you have thoughts on how this can be resolved?
